assignments/prototypes.js

```diff
@@ -7,8 +7,42 @@
   
   Each constructor function has unique properties and methods that are defined in their block comments below:
 */
+  const GameObject = function(attributes){
+    this.createdAt = 'June52019',
+    this.name = attributes.name,
+    this.dimensions = attributes.dimensions
+    this.destroy = function() {
+      return `${this.name} was removed from the game.`}
+  }
+GameObject.prototype.destroy = function () {
+}
   
+  const CharacterStats = function(attributes){
+    GameObject.call(this, attributes)
+    this.healthPoints = attributes.healthPoints
+    this.takeDamage = function() {
+      return `"${this.name} took damage"`}
+    
+    
+
+  }
+CharacterStats.prototype.takeDamage = function () {}
+  
```

Comment:
The other advantage of prototypes is that we can 'inhereit' them from another object. You do it below, but here we want to get all of the GameObject prototypes and apply them to CharacterStats, `CharacterStats.prototype = Object.create(GameObject.prototype)`